static/js/hoc/withMemberForm.js

```diff
@@ -31,21 +43,73 @@ const withMemberForm = (WrappedComponent: *) => {
       }
     }
 
+    componentWillUnmount() {
+      const { endFormEdit } = this.props
+      endFormEdit()
+    }
+
     loadData = async () => {
-      const { channel, loadChannel, loadMembers, members } = this.props
+      const {
+        channel,
+        loadChannel,
+        loadMembers,
+        members,
+        beginFormEdit
+      } = this.props
       if (!channel) {
         await loadChannel()
```

Comment:
Is there a reason we load the channel and then members serially here? Now that we're initializing the form after both of these it is probably worth parallelizing those loads with a `Promise.all`

Reply:
No reason, I'll change it. By the way there are other places where we do this kind of sequential requests, I made an issue to address those: https://github.com/mitodl/open-discussions/issues/966